test/testTrainingOuputEquality.jl

```diff
@@ -50,7 +50,7 @@ som2 = initGigaSOM(dfSom, 10, 10)
 som2.codes = som2Codes
 som2 = trainGigaSOM(som2, dfSom)
 winners2 = mapToGigaSOM(som2, dfSom)
-embed2 = embedGigaSOM(som2, dfSom, k=10, smooth=0.0, adjust=0.5)
+embed2 = embedGigaSOM(som2, dfSom, k=10)
```

Comment:
What if I called `embedGigaSOM(som2, dfSom, k=10, smooth=0.0, adjust=0.5)`? Is this still possible?

Reply:
I added some parameters to the tests for batch/parallel gigasom+embedsom so that "all parameters are checked". No parametrization is actually needed here, so I removed it completely. The `k` is lowered from the default only for the purpose of faster computation and can be omitted as well.

Anyway, the API did not change, I just don't think it's necessary to specify any arguments in this test.

Reply:
Sounds good 👍